File: JAVApb5/src/main/java/concurs/repository/IRepositoryAngajatOficiu.java
package concurs.repository;

import concurs.domain.AngajatOficiu;

import java.util.List;

public interface IRepositoryAngajatOficiu {
   public AngajatOficiu add(AngajatOficiu e);
   public AngajatOficiu delete(AngajatOficiu e);
   public AngajatOficiu update(AngajatOficiu e);
   public AngajatOficiu findOne(Long id);
   public List<AngajatOficiu> findAll();
}

File: CSharp/CSharp/CSharp/repository/IRepositoryAngajatOficiu.cs
using CSharp.domain;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CSharp.repository
{
    interface IRepositoryAngajatOficiu
    {

         AngajatOficiu Add(AngajatOficiu e);
         AngajatOficiu Delete(AngajatOficiu e);
         AngajatOficiu Update(AngajatOficiu e);
         AngajatOficiu FindOne(long id);
         List<AngajatOficiu> FindAll();
    }
}
